src/mento.ts

```diff
@@ -0,0 +1,260 @@
+import { BigNumber, ContractTransaction, Signer } from 'ethers'
+import {
+  IBroker,
+  IBroker__factory,
+  IExchangeProvider,
+  IExchangeProvider__factory,
+} from '@mento-protocol/mento-core-ts'
+import { Provider, TransactionResponse } from '@ethersproject/providers'
```

Comment:
Instead of a separate import for this, a common way would be to import `providers` from ethers and then use `providers.TransactionResponse`, etc.